### DISCUSSION TITLE
How to install airflow through proxy website

### DISCUSSION THREAD

**Discussion by xiaohuihuitadiezcm** [apache/airflow#36786] [Q&A]
### Official Helm Chart version

1.7.0

### Apache Airflow version

2.4.1

### Kubernetes Version

v1.27.3

### Helm Chart configuration

default

### Docker Image customizations

apache/airflow:2.4.1
postgres bitnami/postgresql:11.12.0-debian-10-r44
redis redis:6-bullseye

### What happened

My environment does not have direct access to the internet，but I can access the website behind a proxy，forexample I can access the website "http://9.199.0.6/airflow/" instead of "https://airflow.apache.org/"
when I run the command
helm upgrade --install airflow apache-airflow/airflow --namespace airflow --version 1.7.0  --debug --create-namespace
it can install the pod,service,pvc...,but the "airflow-run-airflow-migrations" step failed.
![image](https://github.com/apache/airflow/assets/124246930/32f2e324-0866-49e8-8970-e37be272d332)

![image](https://github.com/apache/airflow/assets/124246930/36aa691e-9dff-4c0d-b2c2-96608f74b96f)

my apache-airflow-index.yaml

![image](https://github.com/apache/airflow/assets/124246930/e8d4d742-9afd-4e75-b518-b27cbcdba364)

### How to reproduce

Added how to install using helm in offsite situation in official documentation

### Are you willing to submit PR?

- [x] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
First of all do not open airfflow issues for clearly question on how to deal with your infrastructure. You should open discussion (and when you opened issue you were explained to only open issue for "airflow" issue".

Re: your question. You should do it the same way as any other software you install it there. There is no prescribed way, but rather than asking here, I suggest you open an issue to your infrastructure/security team that manages your firewall. They are much better equipped to answer it (and likely paid for the job - unlike free help you got here.

However, in any case you need to find out why your job failed, rather than jump to conclusion that it's because of proxy. Please find out the logs that tell you why your pod is crashing and likely you will have answer there.
